### PR TITLE
Add linux and alexa

### PR DESCRIPTION
**Pull request by erinosher**

### CODE REVIEW

File: src/_events/intro-to-linux/summary.md [hunk 1]
```diff
@@ -0,0 +1 @@
+Get to grips with Linux at our intro to Linux Workshop presented by Primoõz Fabiani
```
Comment (emberdex) [inline src/_events/intro-to-linux/summary.md]: Primož*

File: src/_events/talking-to-alexa/description.md [hunk 2]
```diff
@@ -0,0 +1,7 @@
+# Abstract
+In this talk we look into how we build a conversation with the Amazon Echo and Alexa. We'll look at the design, configuration and (a little) code that goes into building that natural back and forth, how we can use the abilities of the device to enhance the results, and how different APIs can be used to help enhance the experience.
```
Comment (emberdex) [inline src/_events/talking-to-alexa/description.md:2]: Try not to use the word "enhance" twice

File: src/_events/talking-to-alexa/summary.md [hunk 3]
```diff
@@ -0,0 +1 @@
+Learn about how easy and awesome it is to make an app for Alexa during Steven Pears Talking to Alexa talk and workshop.
```
Comment (emberdex) [inline src/_events/talking-to-alexa/summary.md]: Steven Pears'*

**Review by emberdex — CHANGES_REQUESTED** [review]
couple minor things

File: src/_events/talking-to-alexa/description.md (hunk 2 above)
Comment (erinosher) [reply src/_events/talking-to-alexa/description.md:2]: Not my text. So we'll not be changing that.

Comment (emberdex) [reply src/_events/talking-to-alexa/description.md:2]: :+1:

**Review by emberdex — APPROVED** [review]
LGTM

**Pull request merged**